### ISSUE TITLE
1.13 support please

### ISSUE THREAD

**Issue by canhvjp** [Ataraxia1339/CommandGen#1]
1.13 support pls

**Comment by Ataraxia1339** [comment]
Spent a while trying to get it to work for 1.13. Fixed all formatting issues I could find, game registered the command as valid, but placing the command block only got it to summon the falling block, command block minecart and activator rail. There's probably something implemented in later versions of Minecraft that allows all the commands to run, but right now I can't get the passengers of the falling block to execute. Let me know if someone's figured it out, but right now as far as I know it's not possible in 1.13 (there's an older version of the command that creates more falling blocks, but since that relies on passengers to run commands too I doubt that would work). I'll update the readme to reflect this.

tl;dr sorry no can do
